.gitignore

```diff
@@ -5,3 +5,5 @@ build_qurt
 build_bebop
 build_edison
 __pycache__
+/.cproject
```

Comment:
This time I use the eclipse to handler the project. This .cproject  file was created by eclipse. and commit the code by GITHUB plugin, it's new to me. You can ignore this change.